src/components/Block/Block.css

```diff
@@ -27,6 +27,9 @@
 .y-block__topSpacing-xSmall {
   margin-top: $gutterWidth__xSmall;
 }
+.y-block__topSpacing-xxSmall {
```

Comment:
we should remove all the xxSmall changes, 2px doesn't map to our 4px GutterSize (which is going to be replaced by a scalar number in the future)